### ISSUE TITLE
Use of json for policy in authz without a protobuf alternative is obtuse

### ISSUE THREAD

**Issue by haussli** [openconfig/gnsi#136]
Use of json for policy in authz without a protobuf alternative is obtuse.  Why use json at all?  And, why not offer a native protobuf alternative?

My suggestion is that at the very least, a native protobuf version should be supported.  This is not my call though; @tomek-US probably needs to decide this.

This is a split of issues/99.

**Comment by haussli** [comment]
IMHO, support 1 format - protobuf.  json inside protobuf seems unnecessarily kinky.  But, I do not think this is my decision and there is clearly some precedence here with the origin of authz.

**Comment by morrowc** [comment]
I think the discussion in #99 ended up with: "Hey, the gRPC base level code, for reaosns, expects JSON for this, so this service is just implementing that wish."

I dislike json for a ton of reasons, but.... I don't know that we get to change this implementation bit here.
I suppose proposing a change in the gRPC base auth policy is a direction we could pursue.

**Comment by haussli** [comment]
Ja; I understand that the server code has implemented json.  I think that that is its sloppy problem. :)  For consistency, it should convert protobuf to json, if it insists upon using json. 

Some quick searching implies that there are ways to translate protobuf to json, though I do not know how well that works nor if it is specific to java nor if it works in the reverse.  Worth investigating if there is interest in supporting proto instead of json.  Just IMO.
